pkg/template/servicebroker/test-scripts/bind.sh

```diff
@@ -2,8 +2,9 @@
 
 . shared.sh
 
-serviceUUID=${serviceUUID-$(oc get template cakephp-mysql-example -n openshift -o template --template '{{.metadata.uid}}')}
+serviceUUID=${serviceUUID-$(oc get template jenkins-ephemeral -n openshift -o template --template '{{.metadata.uid}}')}
```

Comment:
please can you reset the change here and export serviceUUID to something else in the oc cluster up script?